### DISCUSSION TITLE
Using a collection title template in a shared item page

### DISCUSSION THREAD

**Discussion by mark-james** [directus/directus#15950] [Archive]
Hi,
For a general collection item of type objective, I've set up a title template like the below -
![image](https://user-images.githubusercontent.com/26987752/195311961-25b35792-eae0-499d-b238-9c70d57e4dc7.png)

While when viewing as a anonymous user via a shared link, it's show like this -
![image](https://user-images.githubusercontent.com/26987752/195312084-ecba2170-5af9-457f-b7ff-00d90c2e332e.png)

Is there a way to get the template functionality setup for the anonymous link? Or is this a limitation with the current feature?

thanks
Mark

**Comment by rijkvanzanten** [comment]
I think it's a limitation of the current shared setup. I believe it renders the title as plain text just based on the name of the collection you're viewing, rather than reusing the title template
